great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -130,7 +133,8 @@
 yaml.default_flow_style = False
 
 
-class BaseDataContext(ConfigPeer):
+# TODO: <WILL> Most of the logic here will be migrated to EphemeralDataContext
+class BaseDataContext(EphemeralDataContext, ConfigPeer):
```

Comment:
@Shinnnyshinshin This makes me a little bit hesitant...  Do we really need multiple inheritance here?  Maybe we do -- happy to discuss (avoiding it would be really advantageous, but sometimes we cannot avoid it).  Thanks.